dGame/dComponents/PropertyEntranceComponent.h

```diff
@@ -6,13 +6,20 @@
 #include "eReplicaComponentType.h"
 #include "PropertySelectQueryProperty.h"
 
+enum class ePropertySortType : int32_t {
+	SORT_TYPE_FRIENDS = 0,
```

Comment:
does this need `SORT_TYPE` in front of each value?

Reply:
no it does not.  That can be its own PR if you can take care of that